pkg/datapath/linux/routing/routing_test.go

```diff
@@ -297,3 +285,8 @@ func linkExistsWithMAC(c *C, macAddr mac.MAC) bool {
 
 	return false
 }
+
+func runFuncInNetNS(c *C, run func() error) {
```

Comment:
Could this be removed in favor of `testutils.NewNetNS`/`ns.Do()`?

Reply:
...I forgot that `migrate_test` uses it too, and that's why I left it. Still, not so many refs, will drop.